product_docs/docs/mongo_data_adapter/5/06_features_of_mongo_fdw.mdx

```diff
@@ -96,3 +96,58 @@ Steps for retrieving the document:
     { "_id" : { "$oid" : "58a1ebbaf543ec0b9054585a" }, "warehouse_id" : 2, "warehouse_name" : "Laptop", "warehouse_created" : { "$date" : 1447229590000 } }
     (2 rows)
     ```
+
+## Accessing sub-json
```

Comment:
The feature in NOT a sub feature of "Full-document retrieval". This is the separate feature.

Reply:
Got it.